lib/PaymentRails/ResourceCollection.php

```diff
@@ -40,9 +40,10 @@ class ResourceCollection implements Iterator
      */
     public function  __construct($response, $items, $pager)
     {
-        $this->_page = $response["meta"]["page"];
+        // Add "meta" fields only when they exist in JSON response
+        $this->_page = isset($response["meta"]) ? $response["meta"]["page"] : null;
```

Comment:
which PHP version are we using?
if php7+, we can use null coalescing.`$response["meta"]["page"] ?? null`